community/mei-board/protocols/2020-03-25_Virtual-Board-Meeting.md

```diff
@@ -0,0 +1,90 @@
+	- Maybe a week before the conference
+	- The idea is to have the papers ready in preparation for the conference (this can be indicated in the email). Being able to read the papers beforehand will help presenters to keep their presentations short—now that they have just 3 minutes to present—and go directly to the discussion.
+- A few suggestions about the proceedings:
+	- Word template for ISMIR (is ready)
+	- People to send the PDF not the Word document
+	- Just a set of guidelines in the email asking for the conference papers (font, size, spacing, citation guidelines)
+	- We have only 8 weeks, so we shouldn’t design a template now. Again, we can use a pre-existing Word template (e.g., ISMIR) and send PDF (no editing to be expected).
+
+## MEC 2021
+
+- Stefan Münnich will be the chair for next year
+
+## Development Meeting
+
+- Klaus Rettinghaus hosting it in Leipzig
+- Dates: November Thursday 12 – Saturday 14
+- Very little organization overhead behind this meeting
+
+## Metadata meeting 
+
+- Two- or three-day workshop in Mainz.
+(Officially, we are supposed to meet once a year in Mainz.)
+- Make sense to have it next to the development meeting on Leipzig
+- It is best if these two meetings are kept separated though (not combining them into one)
+- It is better to have the metadata meeting first
+
+So, Metadata Meeting can be Monday 09 – Wednesday 11 in Mainz, and then we move all together in train to Leipzig to have the Development Meeting on Thursday 12 – Saturday 14.
+
+## Editor-in-chief for the Guidelines
+
+- **Johannes Kepper**. He is already the chief editor without the actual title.
+- Maybe it is better for more than one person. **Nomination: Klaus Rettinghaus** as the other editor in chief.
+- Maybe **associate editors, one per chapter**. The ones for each chapter guarantee consistency of that chapter in the guidelines (and also consistency of that chapter with the specs), diluting the chief editors’ burden.
+- This would result in a **hierarchical system**:
+	- Editors in chief (Johannes and Klaus?)
+	- Associate editors = Chapter editors
+	(Maybe each of them with a fork of the guidelines where contributors can work. When reaching an agreement, the chapter editor can make a PR to the original repo for the editors in chief to evaluate.)
+	- Contributors to each chapter
+
+## MEI Guidelines
+
+- Contributors can be listed at the end of each chapter (own section)
+- This section can be added by the “chapter editor” by quickly listing the contributors according to their contributions (see commit history)
+
+**The last two topics can be discussed in further detail in the ODD Friday**
+
+### Copyright
+
+This is for the images used in the guidelines (when they come from manuscripts):
+
+- At the beginning of the guidelines, a statement: “We consider the use of images as fair use.”
+- MEI falls into German law, 15% of copyright work that can be used. So, very small snippets are all right.
```

Comment:
I would not be too sure about the applicable law

Reply:
This is what Johannes said. I can just "cite" it.